crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -1316,6 +1320,93 @@ impl
     }
 }
 
+impl
+    TryFrom<(
+        &CybersourceRouterData<&types::PaymentsAuthorizeRouterData>,
+        Box<hyperswitch_domain_models::router_data::PazeDecryptedData>,
+    )> for CybersourcePaymentsRequest
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        (item, paze_data): (
+            &CybersourceRouterData<&types::PaymentsAuthorizeRouterData>,
+            Box<hyperswitch_domain_models::router_data::PazeDecryptedData>,
+        ),
+    ) -> Result<Self, Self::Error> {
+        let email = item.router_data.request.get_email()?;
+        let (first_name, last_name) = paze_data
+            .billing_address
+            .name
+            .unwrap_or(
+                item.router_data
+                    .get_optional_billing()
+                    .and_then(|address| address.get_optional_full_name())
+                    .ok_or(errors::ConnectorError::MissingRequiredField {
+                        field_name: "billing_address.name",
+                    })?,
+            )
+            .peek()
+            .split_once(' ')
```

Comment:
This logic may be incorrect though, splitting on first space to obtain first name and last name?

Reply:
Should we add an additional check where if `first_name` or `last_name` are empty strings then we can throw missing field error?

Reply:
I was thinking of situations where the beginning of the name contains initials, as in `R. K. Narayan` for example. There would be multiple edge cases honestly.

Do we have provision to accept first name and last name as two separate fields itself? If that's possible, I'd prefer we opt for it. If that's not possible, we may have to consider other options.

Also looping in @Narayanbhat166.

Reply:
We do have the provision to collect `first_name` and `last_name` at hyperswitch which I've fixed.

In case of the billing name coming from paze we only receive full name which is handled by splitting it on the first whitespace for now. Once we start testing on sandbox we can take further decision on how to handle this. This extensive testing is not possible from backend for now because we only have 1 paze payload.